scalableminds/webknossos
# (Backend only) Add project priority to progress report json

### URL of deployed dev instance (used for testing):
- https://progressreportpriority.webknossos.xyz

### Steps to test:
- create some tasks for sampleProject
- go to Statistics→Project Progress, select team, 
- open Browser’s Network Tab
- click Search
- progressOverview reply should contain priority field (set to 100 for sampleProject by default)

### Issues:
- backend part for #3474 

------
